baremaps-geoparquet/src/test/java/org/apache/baremaps/geoparquet/GeoParquetWriterTest.java

```diff
@@ -0,0 +1,150 @@
+import org.locationtech.jts.geom.GeometryFactory;
+import org.locationtech.jts.geom.Point;
+
+class GeoParquetWriterTest {
+
+  @Test
+  @Tag("integration")
+  void testWriteAndReadGeoParquet() throws IOException {
+    // Create the output file
+    Configuration conf = new Configuration();
+    Path outputPath = new Path("target/test-output/geoparquet-test.parquet");
+
+    try {
+      // Define the Parquet schema
+      MessageType type = Types.buildMessage()
+          .required(PrimitiveTypeName.BINARY).as(LogicalTypeAnnotation.stringType()).named("name")
+          .required(PrimitiveTypeName.BINARY).as(LogicalTypeAnnotation.stringType()).named("city")
+          .optional(PrimitiveTypeName.BINARY).named("geometry")
+          .named("GeoParquetSchema");
+
+      // Create GeoParquet metadata
+      Map<String, Column> columns = new HashMap<>();
+      columns.put("geometry", new GeoParquetMetadata.Column(
+          "WKB",
+          List.of("Point"),
+          null,
+          null,
+          null,
+          null));
+
+      GeoParquetMetadata metadata = new GeoParquetMetadata(
+          "1.0",
+          "geometry",
+          columns,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null);
+
+      // Create a Point geometry
+      GeometryFactory geometryFactory = new GeometryFactory();
+      Point point = geometryFactory.createPoint(new Coordinate(1.0, 2.0));
+
+      // Create the GeoParquetWriter
+      try (ParquetWriter<GeoParquetGroup> writer = GeoParquetWriter.builder(outputPath)
+          .withType(type)
+          .withGeoParquetMetadata(metadata)
+          .build()) {
+
+        // Create a GeoParquetGroup and write it
+        GeoParquetSchema geoParquetSchema =
+            GeoParquetGroupFactory.createGeoParquetSchema(type, metadata);
+        GeoParquetGroup group =
+            new GeoParquetGroup(type.asGroupType(), metadata, geoParquetSchema);
+        group.add("name", "Test Point");
+        group.add("city", "Test City");
+        group.add("geometry", point);
+
+        // Write the group
+        writer.write(group);
+      }
+
+      // Now read back the file using GeoParquetReader
+      GeoParquetReader reader = new GeoParquetReader(outputPath, null, conf);
+      GeoParquetGroup readGroup = reader.read().findFirst().orElse(null);
+
+      assertNotNull(readGroup, "Read group should not be null");
+
+      // Verify the data
+      assertEquals("Test Point", readGroup.getStringValue("name"));
+      assertEquals("Test City", readGroup.getStringValue("city"));
+
+      Point readPoint = (Point) readGroup.getGeometryValue("geometry");
+      assertEquals(point.getX(), readPoint.getX(), 0.0001);
+      assertEquals(point.getY(), readPoint.getY(), 0.0001);
+    } finally {
+      outputPath.getFileSystem(conf).delete(outputPath, false);
+    }
+  }
+
+  @Test
+  @Tag("integration")
+  void copy() throws IOException {
```

Comment:
Add one assertion in this test. And change method name

Reply:
Ok, I added a comment stating that we don't need access to hadoop's config parameters for now.

Reply:
Thanks, I checked the number of rows in the copied file